huaweicloud/services/acceptance/dc/resource_huaweicloud_dc_virtual_interface_test.go

```diff
@@ -189,3 +243,33 @@ resource "huaweicloud_dc_virtual_interface" "test" {
 }
 `, testAccVirtualInterface_base(name), acceptance.HW_DC_DIRECT_CONNECT_ID, name, vlan)
 }
+
+func testAccVirtualInterface_acrossTenant(name string, vlan int) string {
+	return fmt.Sprintf(`
+%[1]s
+
+resource "huaweicloud_dc_virtual_interface" "test" {
+  direct_connect_id = "%[2]s"
+  vgw_id            = "%[3]s"
```

Comment:
Please using a resource `huaweicloud_dc_virtual_gateway` to replace environment variable.